### DISCUSSION TITLE
To get total energy for the given structure using pretrained model

### DISCUSSION THREAD

**Discussion by Harpriyaminhas** [CederGroupHub/chgnet#198] [Announcements]
Can we get the total energy (eV) of a system using the pre-trained CHGNet model instead of predicting the energy (eV/atom)?

**Comment by bowen-bd** [comment]
You can multiply by the number of atoms

**Reply by janosh** [reply]
worth mentioning that that'll give you the MP2020 corrected total energy which CHGNet was trained on. you need to subtract the MP2020 energy corrections to get the raw PBE energy
